### DISCUSSION TITLE
路边社對2024年的展望：討論明年會發生什麼重大經濟和社會事件

### DISCUSSION THREAD

**Discussion by Lewisrxliu** [RoadSideUniversity/ClubHouse2024#47] [General]
感谢路边
https://www.clubhouse.com/house/%E8%B7%AF%E9%82%8A%E7%A4%BE-economic-topics-discussion
12.15.2023
https://www.clubhouse.com/room/WXphy5ew5tdqAqRJ:k8pLOlCc5yQfNEgny5qJYoATs7VyBtKRfgII1mMH5N8?utm_medium=ch_instant_invite&utm_campaign=lpYfcJN9MC3YSlNJ-t3K_A-1026398

**Comment by Lewisrxliu** [comment]
知识圈和知识同时下行，世界级全面弱智的时代已经来临

https://twitter.com/cskun1989/status/1736720900446146883?s=52&t=prwirIqnuft51wSJpRmNxw

https://finance.sina.cn/2023-03-12/detail-imykratx9190179.d.html

Ref:《未来30年》 郑永年

**Comment by Lewisrxliu** [comment]
CH2024的社区实践
https://too.craft.me/vhWNCMkLkgzvIw

**Comment by guttasingapore** [comment]
2025年的12月，本该展望2026的时候，一份3年前的报告突然伴随着垃圾浮现在眼前， 让我想起了2023年底这些大佬们一本正经的胡说八道！

它们可能是故意的, 现在也许还是一样

https://am.jpmorgan.com/sg/en/asset-management/per/insights/market-insights/

![image](https://github.com/user-attachments/assets/c90ac9a6-4290-43ac-9b96-1b73f07667d4)

**Comment by guttasingapore** [comment]
2026年的展望，不需要多久，我觉得很快就会发现：人类预测未来有多么不靠谱

https://www.dbresearch.com/PROD/RPS_EN-PROD/PROD0000000000611716/Anything_but_dull.PDF

[Anything_but_dull.PDF](https://github.com/user-attachments/files/24126902/Anything_but_dull.PDF)
